### PR TITLE
Cleanup Documentation and bash completions

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

File: transfer.md [hunk 1]
```diff
@@ -23,29 +23,31 @@ For many troubleshooting and information collection steps, there may be an exist
...
+| :--- | :--- |
+| `docker run`  | [`podman run`](./docs/podman-run.1.md) |
 | `docker exec` | [`podman exec`](./docs/podman-exec.1.md) |
 | `docker info` | [`podman info`](./docs/podman-info.1.md)  |
 | `docker inspect` | [`podman inspect`](./docs/podman-inspect.1.md)       |
 | `docker logs` | [`podman logs`](./docs/podman-logs.1.md)                 |
-| `docker ps` | [`podman ps`](./docs/podman-ps.1.md) |
-| `docker stats` | [`podman stats`](./docs/podman-stats.1.md)|
+| `docker ps`   | [`podman ps`](./docs/podman-ps.1.md) |
+| `docker stats`| [`podman stats`](./docs/podman-stats.1.md)|
```
Comment (TomSweeneyRedHat) [inline transfer.md]: should there be a space here between the ' and the | after the first stats?

Comment (TomSweeneyRedHat) [reply transfer.md]: Nope, I see, you're keeping the |'s in line.  Ignore.

File: transfer.md [hunk 2]
```diff
@@ -58,36 +60,38 @@ There are other equivalents for these tools
...
+| `docker port`     ||
+| `docker rename`   | podman does not support rename, you need to podman rm/podman create to rename a container.|
+| `docker restart`  | podman does not support restart.  We recommend that you put your podman containers into a systemd unit file and use it for restarting applications.|
+| `docker search`   ||
+| `docker secret`   ||
+| `docker service`  ||
+| `docker stack`    ||
+| `docker swarm`    | podman does not support swarm.  We support Kubernetes for orchestration using [CRI-O](https://github.com/kubernetes-incubator/cri-o).|
+| `docker system`   ||
+| `docker volume`   | podman does not support volumes.  Volumes should be built on the host operating system and then volume mounted into the containers.|
```
Comment (TomSweeneyRedHat) [inline transfer.md]: Can we add "See podman mount for more information." or something to that effect?

File: transfer.md [hunk 3]
```diff
@@ -58,36 +60,38 @@ There are other equivalents for these tools
...
+| Missing command | Description|
+| :--- | :--- |
+| `docker container`||
+| `docker events`   ||
+| `docker image`    ||
+| `docker network`  ||
+| `docker node`     ||
+| `docker plugin`   |podman does not support plugins.  We recommend you use alternative OCI Runtimes or OCI Runtime Hooks to alter behaviour of podman.|
+| `docker port`     ||
+| `docker rename`   | podman does not support rename, you need to podman rm/podman create to rename a container.|
```
Comment (TomSweeneyRedHat) [inline transfer.md]: suggest "need to podman" to "need to use podman"

File: transfer.md (hunk 2 above)
Comment (rhatdan) [reply transfer.md]: Well this is not about podman mount, more about 
```
podman run -v /source:/dest fedora sh
```
In the docker world they defined special Volume plugins, that allow you to execute things like

```
docker run -v PLUGIN:/DEST fedora sh
```
Which will do not support

File: transfer.md (hunk 3 above)
Comment (rhatdan) [reply transfer.md]: Fixed

**Comment by rhatdan** [comment]
Since I fixed the one NIT we agree on, I am going to merge.  We can work with future PRs to further clean up the documentation.
@rh-atomic-bot r=TomSweeneyRedHat

**Pull request closed without merging**